staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go

```diff
@@ -1838,9 +1852,24 @@ func TestGetTable(t *testing.T) {
 		item       bool
 	}{
 		{
-			accept:     runtime.ContentTypeJSON + ";as=Table;v=v1;g=meta.k8s.io",
+			accept:     runtime.ContentTypeJSON + ";as=Table;v=v1alpha1;g=meta.k8s.io",
 			statusCode: http.StatusNotAcceptable,
 		},
+		{
+			item:   true,
+			accept: runtime.ContentTypeJSON + ";as=Table;v=v1;g=meta.k8s.io",
+			expected: &metav1beta1.Table{
```

Comment:
Shouldn't this be metav1.Table?

Reply:
LGTM if this is resolved.

Reply:
reflect.deepequal on an aliased type reports no diff, but I'll change it

Reply:
ah, it's aliased? I missed that. It's still probably good to be clear.